src/lib/y2storage/disk_analyzer.rb

```diff
@@ -223,120 +173,97 @@ def mbr_gap
     # @see #scope
     #
     # @return [Hash{String => Array<Partition>}] see {#partitions_with_id}
-    def windows_partitions
-      @windows_partitions ||= begin
-        result = find_windows_partitions
-        log.info("Found Windows partitions: #{result}")
-        result
-      end
+    def windows_partitions(*disks, limit: DEFAULT_DISKS_LIMIT)
```

Comment:
Nevermind, I actually messed up limit of checks/mounts (that is related to the number of partitions) with the number of passed disks. It's fine as it is, we just need a better name for `:limit`